tools/targets/GenerateAssemblyInfo.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please revert all "tools" files related changes that are not part of requested change.

Reply:
done.

Reply:
done.